src/govuk/components/exit-this-page/_index.scss

```diff
@@ -78,4 +78,8 @@
     left: 0;
     background-color: govuk-colour("white");
   }
+
+  .govuk-exit-this-page-hide-content *:not(.govuk-exit-this-page__overlay) {
+    display: none;
```

Comment:
Also curious about whether to stick `!important` here or not. 

The specificity seems high enough to not need it, assuming pages are structured how we like 'em, but this does feel like one of those 'absolutely override this in all circumstances' situations that `!important` exists specifically for.

Might want to ensure that it works with our display override classes (`.govuk-!-display-display`, `.govuk-!-display-inline`, etc.), though the higher specificity makes me assume it does.

Reply:
From a bit of testing I've found the following I've found that we _do_ need `!important` when we're on a page with display override classes and that the specificity overrides the competing `!important` statements. It makes sense for it to be here and doesn't have any other costs that I can see so it's going in.